cmd/server/server.go

```diff
@@ -26,15 +26,17 @@ import (
 	ot "github.com/opentracing/opentracing-go"
 	"github.com/spf13/cobra"
 
-	"github.com/golang/protobuf/ptypes/struct"
+	"istio.io/mixer/adapter"
+	"istio.io/mixer/pkg/adapterManager"
 	"istio.io/mixer/pkg/api"
+	"istio.io/mixer/pkg/aspect"
 	"istio.io/mixer/pkg/attribute"
+	"istio.io/mixer/pkg/config"
 	"istio.io/mixer/pkg/tracing"
 
-	denyadapter "istio.io/mixer/adapter/denyChecker"
-	"istio.io/mixer/pkg/aspect"
+	"time"
```

Comment:
this looks like it needs cleanup... i would expect std packages to be grouped, etc.